### PR TITLE
[webapi] Add "-manual" suffix for backgrounds manual tests

### PR DESCRIPTION
**Pull request by JianfengXu**
Impacted TCs num(approved): New 0, Update 56, Delete 0
Unit test Platform: Crosswalk Project for Android 15.44.364
Unit test result summary: Pass 55, Fail 1, Blocked 0

https://crosswalk-project.org/jira/browse/XWALK-4502

### CODE REVIEW

File: webapi/tct-backgrounds-css3-tests/tests.full.xml [hunk 1]
```diff
@@ -6551,7 +6551,7 @@
       </testcase>
       <testcase component="W3C_HTML5 APIs/DOM, Forms and Styles/CSS Backgrounds and Borders Module Level 3" execution_type="manual" id="background-origin-002" priority="P2" purpose="Check if background-origin with initial value display correctly in visual" status="approved" type="compliance">
         <description>
-          <test_script_entry timeout="90">/opt/tct-backgrounds-css3-tests/backgrounds/csswg/background-origin-002.html</test_script_entry>
+          <test_script_entry timeout="90">/opt/tct-backgrounds-css3-tests/backgrounds/csswg/background-origin-002-manual.html</test_script_entry>
```
Comment (zqzhang) [inline webapi/tct-backgrounds-css3-tests/tests.full.xml]: Not sure why you add '-manual' to the reftests testing files.

Comment (JianfengXu) [inline webapi/tct-backgrounds-css3-tests/tests.full.xml]: Run  ref  tests automatically need webdriver , if no webdriver, we must run these manually. Same as bdd tests.

Comment (zqzhang) [inline webapi/tct-backgrounds-css3-tests/tests.full.xml]: @jennycao @yugang 

These tests, I think, are running on both nightly testing system (with webdriver support) and web testing service (without webdriver). Right?

If yes, I don't think this `-manual` suffix is acceptable. What do you think?

**Comment by zqzhang** [comment]
Talked to @yugang offline with the following clearified:

- The "-manual" suffix is ONLY used by web testing service for manual tests identification.
- The web driver based nightly testing uses `<refer_test_script_entry>` in tests.xml to run reftests, whatever the file names in `<test_script_entry>` are suffixed "-manual" or not.
- From the pull request changes, we can see that all these tests have been recorded as `execution_type="manual"`. That means these changes shall have no impact on current being test suite release and coverage analysis.

Therefore, I will merge it.

**Pull request merged**